### ISSUE TITLE
Error when changing fen position while creating it.

### ISSUE THREAD

**Issue by ezz3r** [shaack/cm-chessboard#134]
"Cannot read properties of undefined (reading 'toUpperCase')"

` const board = new Chessboard(document.getElementById("board"), {
        position: "PPPPPPPP/PPPPPPPP/00000000/00000000/00000000/00000000/PPPPPPPP/PPPPPPPP",
        assetsUrl: "./css/",
        style: {borderType: BORDER_TYPE.none, pieces: {file: "pieces/staunty.svg"}, animationDuration: 300},
        orientation: playercolor,
        extensions: [
            {class: Markers, props: {autoMarkers: MARKER_TYPE.square}},
            {class: PromotionDialog},
            {class: Accessibility, props: {visuallyHidden: true}}
        ]
    })`

**Comment by shaack** [comment]
I think these "0" are not part of a valid FEN. See https://en.wikipedia.org/wiki/Forsyth–Edwards_Notation

**Comment by ezz3r** [comment]
Yes thats the problem, so i have to type 8 instead of the zeros. Is there a way i can still do it with 0? 
Because it would be much work to change my database reader etc.
